nodeJS/authentication/Authentication.md

```diff
@@ -23,17 +23,17 @@ By the end of this lesson, you should be able to do the following:
 
 ### Set Up
 
-We're going to be using another Mongo database, so before we begin login to your mongo provider and create a new database and save it's url string somewhere handy.
+We're going to be using another Mongo database, so before we begin login to your mongo provider and create a new database and save its URL string somewhere handy.
 
 To begin, let's set up a very minimal express app with a single MongoDB model for our users. Create a new directory and use `npm init` to start the package.json file then run the following to install all the dependencies we need:
 
 ~~~
 npm install express express-session mongoose passport passport-local ejs
 ~~~
 
-Next let's create our `app.js`:
+Next, let's create our `app.js`:
 
-**IMPORTANT NOTE**: For the moment we are saving our users with just a plain text password.  This is a _really_ bad idea for any real-world project. At the end of this lesson you will learn how to properly secure these passwords using bcrypt. Don't skip that part.
+**IMPORTANT NOTE**: For the moment we are saving our users with just a plain text password.  This is a _really_ bad idea for any real-world project. At the end of this lesson, you will learn how to properly secure these passwords using bcrypt. Don't skip that part.
```

Comment:
I think this comma is unnecessary.

Reply:
The comma would be grammatically correct.

Reply:
il add it back